workspaces/quay/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@internal/quay",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "18 || 20"
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "tsc:full": "tsc --skipLibCheck true --incremental false",
```

Comment:
```suggestion
    "tsc:full": "tsc --skipLibCheck false --incremental false",
```

Not sure why this was changed

Reply:
@awanlin  --skipLibCheck needs to be true the CI to pass